src/App/index.js

```diff
@@ -50,16 +50,15 @@ const Utilities = React.lazy(() => import(/* webpackChunkName: "utilities.chunk"
 
 const App = () => {
 
-    const [version, setVersion] = useState();
+    const [version, setVersion] = useState(undefined);
 
-    const VersionTopBanner = () => {
-
-        useEffect(() => {
-            fetch("https://design.swedbankpay.com/latestVersion.json").then(data => data.json())
-                .then(data => setVersion(data.latestVersion))
-                .catch(error => console.warn("Could not fetch latest version from Azure:", error));
-        }, []);
+    useEffect(() => {
+        fetch("https://design.swedbankpay.com/latestVersion.json").then(data => data.json())
+            .then(data => setVersion(data.latestVersion))
+            .catch(error => console.warn("Could not fetch latest version from Azure:", error));
+    }, []);
```

Comment:
should we move this useEffect to the other useEffect of App (the one of `topbar.init()` ), since they both only run once they both have an empty dependency array?

also, the VersionToBanner React component should be declared outside of App component. It's not a big deal now since it is anyway not used outside of App component, but I think it's best practice.